src/main/java/com/hobbycollection/HobbyCollectionRESTController.java

```diff
@@ -2,6 +2,8 @@
 
 import com.hobbycollection.dto.Collection;
 import com.hobbycollection.service.ICollectionService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
importing classes for logging.
Sources:
https://www.youtube.com/watch?v=cMVj0yF5x0Q

Reply:
Good idea, though this duplicates a previous code review.  When that occurs, first in wins.